src/Mod/Spreadsheet/Gui/SheetTableView.cpp

```diff
@@ -188,55 +188,48 @@ SheetTableView::SheetTableView(QWidget* parent)
         menu.exec(QCursor::pos());
     });
 
-    actionProperties = new QAction(tr("Properties…"), this);
+#define createAction(act, iconPath, text, fun)                                                     \
+    act = new QAction(QIcon {QStringLiteral(iconPath)}, tr(text), this);                           \
+    connect(act, &QAction::triggered, this, fun);                                                  \
+    contextMenu.addAction(act);
```

Comment:
Linter here is absolutely right. This should not be macro but a function - local lambda perhaps?
```suggestion
const auto createAction = [this, &contextMenu](std::string icon, QString text, auto callback) {
    QIcon icon {QString::fromStdString(iconPath)};

    auto* action = new QAction(icon, text, this);
    connect(act, &QAction::triggered, this, callback);
    
    contextMenu.addAction(action);
}
```

The only change you would need to do on call-site would be to wrap every translatable string into `tr()` function, but you should do it anyway to show that this is translatable string not only to compiler but also to developers.